apps/webstore/src/app/pages/auth/profile/profile.component.ts

```diff
@@ -45,25 +48,10 @@ export class ProfileComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    try {
-      this.isLoading = true;
-      this.userService.getUser().subscribe((user: IUser) => {
-        this.currentUser = user;
-        if (this.isVerified !== user.isVerified) {
-          this.verifyService.setIsVerified(user.isVerified);
-        }
-        this.updateFormValues();
-        this.isLoading = false;
-      });
-    } catch (error) {
-      console.error(error);
-      // TODO: handle failed fetching of the data
-    }
-
     this.accountInfoForm = new FormGroup({
       name: new FormControl('', [
         Validators.maxLength(50),
-        Validators.pattern("^[a-zA-Z-' ]*$"),
+        Validators.pattern("^[a-zA-Z-' ÆØÅæøå]*$"),
```

Comment:
We shouldn't verify that the names contain valid characters etc
It is way too much work to handle all possibilities (Do you want to add EVERY SPECIAL CHARACTER out there? Example: ´ąółżęć´ just for Poland)
```suggestion
```
This comment also applies to the city and address